apps/web/public/static/locales/en/common.json

```
@@ -1696,6 +1696,8 @@
   "only_show_first_available_slot_description": "This will limit your availability for this event type to one slot per day, scheduled at the earliest available time.",
   "limit_total_booking_duration": "Limit total booking duration",
   "limit_total_booking_duration_description": "Limit total amount of time that this event can be booked",
+  "booker_booking_limit": "Booker booking limit",
+  "booker_booking_limit_description": "Limit the number of bookings a booker can make for this event type",
```

```suggestion
  "booker_booking_limit_description": "Limit the number of active bookings a booker can make for this event type",
```

make similar changes (active bookings) everywhere else too.